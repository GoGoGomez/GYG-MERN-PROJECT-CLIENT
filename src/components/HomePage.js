import React from 'react';
import { Link } from 'react-router-dom'
// import Background from '../images/home-background.jpg';

// var sectionStyle = {
//   width: "100%",
//   height: "400px",
//   backgroundImage: `url(${Background})`
// };

// class Section extends Component {
//   render() {
//     return (
//       <section style={ sectionStyle }>
//       </section>
//     );
//   }
// }
const HomePage = () => (
  <div className="Homepage-background">
    <div className="Homepage">
    <h1>Welcome to Guzman y Gomez Kawana Catering Page</h1><br />
    <Link to="/menu" className="MenuButton">Start your order</Link>
    
    </div>
  </div>
);

export default HomePage;
