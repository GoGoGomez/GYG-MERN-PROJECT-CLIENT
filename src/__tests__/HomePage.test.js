import React from "react";
import renderer from "react-test-renderer";
import HomePage from "../components/HomePage";
import { MemoryRouter as Router, withRouter } from "react-router-dom";

test("HomePage renders correctly", () => {
  const component = renderer.create(<Router><HomePage /></Router>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});