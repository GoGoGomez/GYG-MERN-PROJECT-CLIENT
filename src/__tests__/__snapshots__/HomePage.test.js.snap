// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<div
  className="Homepage-background"
>
  <div
    className="Homepage"
  >
    <h1>
      Welcome to Guzman y Gomez Kawana Catering Page
    </h1>
    <br />
    <a
      className="MenuButton"
      href="/menu"
      onClick={[Function]}
    >
      Start your order
    </a>
  </div>
</div>
`;
